services/squareone/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Default values for squareone.
+#
+# Global parameters will be set by parameters injected by Argo CD and should
+# not be set in the individual environment values files.
+global:
+  # -- Base URL for the environment
+  # @default -- Set by Argo CD Application
+  baseUrl: ""
+
+  # -- Host name for ingress
+  # @default -- Set by Argo CD Application
+  host: ""
+
+  # -- Base path for Vault secrets
+  # @default -- Set by Argo CD Application
+  vaultSecretsPathPrefix: ""
+
+# -- Number of squareone pods to run in the deployment.
+replicaCount: 1
+
+image:
+  # -- Squareone Docker image repository
+  repository: lsstsqre/squareone
+
+  # -- Image pull policy (tip: use Always for development)
+  pullPolicy: IfNotPresent
+
+  # -- Overrides the image tag.
+  # @default -- Chart's appVersion
+  tag: ""
+
+# -- Secret names to use for all Docker pulls
+imagePullSecrets: []
+
+# -- Overrides the base name for resources
+nameOverride: ""
+
+# -- Overrides the full name for resources (includes the release name)
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use.
+  # @default -- Generated using the fullname template
+  name: ""
+
+# -- Annotations for squareone pods
+podAnnotations: {}
+
+service:
+  # -- Type of service to create
+  type: ClusterIP
+
+  # -- Port of the service to create and map to the ingress
+  port: 80
+
+ingress:
+  # -- Enable ingress
+  enabled: true
+
+  # -- Additional annotations to add to the ingress
+  annotations: {}
+
+  tls: []
```

Comment:
This would become just `tls: false` (or true, whatever makes more sense as a default).